packages/mobile/src/screens/edit-track-screen/EditTrackNavigator.tsx

```diff
@@ -65,9 +68,7 @@ export const EditTrackNavigator = (props: EditTrackNavigatorProps) => {
         <Stack.Screen name='LicenseType' component={LicenseTypeScreen} />
         <Stack.Screen name='KeyBpm' component={KeyBpmScreen} />
       </Stack.Navigator>
-      <GatedContentUploadPromptDrawer
-        isUpload={!props.initialValues.track_id}
-      />
+      <GatedContentUploadPromptDrawer isUpload={!initialValues.track_id} />
```

Comment:
do we have `initialValues.isUpload` now?

Reply:
oh yeah good looks, i kinda want to remove this thing entirely too